Source/relay/TourGuide/Items of the Month/2023/Closed Circuit Pay Phone.ash

```diff
@@ -0,0 +1,87 @@
+RegisterTaskGenerationFunction("IOTMClosedCircuitPayPhoneGenerateTasks");
+void IOTMClosedCircuitPayPhoneGenerateTasks(ChecklistEntry [int] task_entries, ChecklistEntry [int] optional_task_entries, ChecklistEntry [int] future_task_entries)
+{
+    if (!lookupItem("closed-circuit pay phone").have())
+        return;
+
+    string [int] affinityDescription;
+    string [int] questDescription;
+    string url = "inv_use.php?pwd=" + my_hash() + "&which=3&whichitem=11169";
+
+    int shadowLodestones = available_amount($item[Rufus's shadow lodestone]);
```

Comment:
i don't know if i love this being above all the stuff below. doesn't this lead to a weird tile format when you're at loadstone > 0 and rufus quest done? i am not opposed to it just feel like we should rethink it slightly -- also, when in_run = true, shouldn't we try to advise the user not to pick the quests that suck and instead pick the artifact quest that's pretty easy?

Reply:
I'm not sure I see the issue. You have to intentionally set yourself up to be at mod%11 encounters before turning in an item quest to get a lodestone, which is a situation most people will not be in.
As for quest picking, I think there are too many variables (player ability, shinies, etc) to make a truly informed decision about what quest they should take, the bosses (which are now free) drop more quest reagents which makes a positive loop, which is why I don't think advising artifact is helpful.